src/test/java/org/folio/inventory/storage/external/ExternalItemCollectionExamples.java

```diff
@@ -28,16 +24,16 @@
 import org.junit.Before;
 import org.junit.Test;
 
-public class ExternalItemCollectionExamples {
+import lombok.SneakyThrows;
```

Comment:
Could we place this import statement outside the class.

Reply:
Hmm, having investigated this. The import is outside of the class.

I think this is an artefact of how the diff is presented. The diff includes both the addition of the import and a change to the class line. The line above the import in this presentation is the removal of the old class definition.

The code in the branch is:
```
import lombok.SneakyThrows;

public class ExternalItemCollectionExamples extends ExternalStorageTests {
```